include/catch2/catch.hpp

```diff
@@ -0,0 +1,2 @@
+// see https://github.com/catchorg/Catch2/blob/v3.0.1/docs/migrate-v2-to-v3.md#how-to-migrate-projects-from-v2-to-v3
```

Comment:
Adding the file to the include folder here, will mean that it gets packaged and deployed with SI
Since the file is only used for testing please move it to the test folder or change the includes in the tests to directly point to the catch_app.hpp

Reply:
OK